openshift-hack/commitchecker/validate.go

```diff
@@ -19,6 +20,46 @@ var (
 	}
 )
 
+// ValidateReleaseBranchConsistency outputs an error in case we're missing a commit in
+// any of the following (n+2) release branches. This helps us to understand if we're merging a fix that isn't present
+// in later versions of OCP yet - which might cause regression problems in cluster upgrades.
+// This currently only works on release-4.x branches for simplicity.
+func ValidateReleaseBranchConsistency(commits []Commit) []string {
+	var allErrors []string
+
+	branch, err := CurrentBranch()
+	if err != nil {
+		return append(allErrors, fmt.Sprintf("Could not retrieve branch information: %v", err))
+	}
+
+	if IsReleaseBranch(branch) {
```

Comment:
reverse this condition, `if !IsReleaseBranch(branch) return` makes the code easier to read and less indentation.

Reply:
good one, fixing

Reply:
Yeah, that's very useful pattern to fail fast and try to keep the code as "flat" as possible, since each indentation introduces another execution path which in the end lead to unintended complexity.